aced_submission/grip_load.py

```diff
@@ -237,23 +237,27 @@ def delete_project(grip_service: str, graph_name: str, project_id: str, output:
     return json_data
 
 
-def get_project_data(grip_service: str, graph_name: str, project_id: str, output: dict, access_token: str) -> Generator[dict, None, None]:
+def get_project_data(grip_service: str, graph_name: str, project_id: str, output: dict, access_token: str, chunk_size: int) -> Generator[dict, None, None]:
     """Retrieves all of the data for a given project id on a given graph"""
     response = requests.get(
         f"http://{grip_service}:8201/{NGINX_PATH}/{graph_name}/get-vertices/{project_id}",
         headers={"Authorization": f"bearer {access_token}"}
     )
 
-    def stream_res(response):
-        for result in response.iter_lines(chunk_size=None):
-            try:
-                result_dict = orjson.loads(result.decode())
-            except Exception as e:
-                print("Failed to decode: %s", result)
-                raise e
-            yield result_dict["data"]
-
-    return stream_res(response)
+    for result in response.iter_lines(chunk_size=chunk_size):
```

Comment:
Makes sense to pass in the chunk size, but shouldn't we just let it default to something? It seems to default to 512 [when chunk size isn't set](https://github.com/psf/requests/blob/c54d0347e24f38b40881f1e0a8bfc9fbce583298/src/requests/models.py#L858), that could be out main problem is just that it was hardcoded set to None

Reply:
I designed it this way so that we don't have to cut a new release of gen3 submission everytime we want to change the chunk size.